ragaai_catalyst/tracers/agentic_tracing/utils/llm_utils.py

```
@@ -45,6 +45,11 @@ def extract_model_name(args, kwargs, result):
             result = result.to_dict()
             if 'model_version' in result:
                 model = result['model_version']  
+    try:
+        if not model:
+            model = result.raw.model
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)